app/scripts/modules/appengine/helpContents/appengineHelpContents.ts

```diff
@@ -45,6 +45,14 @@ module(APPENGINE_HELP_CONTENTS_REGISTRY, [HELP_CONTENTS_REGISTRY])
                 file here, you should <b>not</b> specify the file path to an <code>app.yaml</code> above.<br></p>
                 <p>If this is a pipeline stage, you can use Spinnaker Pipeline Expressions here.</p>`,
       },
+      {
+        key: 'appengine.serverGroup.configFilesRequired',
+        value: `<p>The contents of an App Engine config file (e.g., an <code>app.yaml</code> or
```

Comment:
I might mention that an `app.yaml` is definitely required here. 

The piece about `should not conflict with the config filepaths above` I think is no longer relevant, since you're hiding that field.

Reply:
:+1: done & added a note about runtime: custom being required as well.